src/resources/platform-adaptations.md

```diff
@@ -739,7 +739,73 @@ Scaffold(
                   .toList(),
             ));
 ```
+### Text fields
 
+Since Android 12, the default UI of text fields follows the design 
+guidelines defined in [Material 3][m3-text-field] (M3). 
+On iOS, an equivalent component is defined in 
+[Human Interface Guidelines][hig-text-field] (HIG). 
+
+<div class="container">
+  <div class="row">
+    <div class="col-sm text-center">
+      <figure class="figure">
+        <img src="/assets/images/docs/platform-adaptations/m3-text-field.png" 
+        class="figure-img img-fluid rounded" alt="Text Field in Material 3" />
+        <figcaption class="figure-caption">
+          Text Field in Material 3
+        </figcaption>
+      </figure>
+    </div>
+    <div class="col-sm">
+      <figure class="figure text-center">
+        <img src="/assets/images/docs/platform-adaptations/hig-text-field.png" 
+        class="figure-img img-fluid rounded" alt="Text Field in Human Interface Guidelines" />
+        <figcaption class="figure-caption">
+          Text Field in HIG
+        </figcaption>
+      </figure>
+    </div>
+  </div>
+</div>
+
+Since text fields require user input,  
+their design should follow platform conventions. 
+
+One way to implement a platform-specific `TextField` 
+in Flutter is to adapt the styling of the 
+Material `TextField`.
```

Comment:
**nit:** Write in active voice per [GSG](https://developers.google.com/style/voice). Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure).
```suggestion
To implement a platform-specific `TextField` 
in Flutter, you can adapt the styling of the 
Material `TextField`.
```